models/CoronSegmentator/scripts/heart_digital_twin.py

```
@@ -0,0 +1,104 @@
+            destination_path = BUNDLE_ROOT / rel_path
+            destination_dir = Path(destination_path).parent
+            destination_dir.mkdir(parents=True, exist_ok=True)
+            download_url(url, filepath=destination_path, hash_val=expected_hash)
+
+        logger.info(f"Starting processing of {self.nii_file}")
+        nii_path = self.input_image
+        seg_folder = create_output_folder(self.output_dir, self.nii_file)
+        try:
+            run_coronary_artery_segmentation(nii_path, seg_folder)
+            return True
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)